tree/dataframe/test/dataframe_definepersample.cxx

```diff
@@ -14,6 +14,16 @@
 #include <atomic>
 #include <memory>
 #include <thread> // std::thread::hardware_concurrency
+#include <vector>
+
+template <typename T>
+void EXPECT_VEC_EQ(const std::vector<T> &v1, const std::vector<T> &v2)
```

Comment:
If possible, I would use a different name for this function to make sure it's not from gtest (or to avoid conflicts if gtest introduces it at some point).